html/user/lookup_account.php

```diff
@@ -72,16 +80,28 @@
     // if no password set, set password to account key
     //
     if (!strlen($user->passwd_hash)) {
-        $user->passwd_hash = $auth_hash;
+        $user->passwd_hash = password_hash($auth_hash , PASSWORD_DEFAULT);
         $user->update("passwd_hash='$user->passwd_hash'");
```

Comment:
[Here](https://github.com/BOINC/boinc/pull/2401/files#diff-2ebf2ee712812c0682531ca5ebf7c713R73) you escape the password hash before its use as part of an SQL clause. If you think it's required, it should be done here as well.

Reply:
After reviewing, the hash does not need to be escaped so I have removed it from the referenced line.